catalog/datasets/muziekschatten-onderwerpen.jsonld

```diff
@@ -0,0 +1,34 @@
+{
+  "@context": "https://schema.org/docs/jsonldcontext.jsonld",
+  "@id": "http://data.muziekschatten.nl/onderwerpen",
```

Comment:
This is made up, isn’t it? Perhaps https://data.muziekschatten.nl/#onderwerpen is better because at least that resolves to some URL.